cli/src/utils/notifications.ts

```diff
@@ -28,12 +28,12 @@ export async function fetchKilocodeNotifications({
 		return []
 	}
 
-	if (!kilocodeToken) {
+	if (!kilocodeToken || typeof kilocodeToken !== "string") {
 		logs.debug("No kilocode token found, skipping notification fetch", "fetchKilocodeNotifications")
 		return []
 	}
 
-	const url = getKiloUrlFromToken("https://api.kilocode.ai/api/users/notifications", kilocodeToken)
+	const url = getKiloUrlFromToken("https://api.kilocode.ai/api/users/notifications", kilocodeToken as string)
```

Comment:
I don't think this `as string` is needed? The `typeof` above should cover this, right?

Reply:
Removed now that we have proper types on the providersCOnfig